src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -251,19 +322,12 @@ impl A320Hydraulic {
         rat_and_emer_gen_man_on: &impl EmergencyElectricalRatPushButton,
         emergency_elec_state: &impl EmergencyElectricalState,
     ) {
-        let min_hyd_loop_timestep =
-            Duration::from_millis(Self::HYDRAULIC_SIM_TIME_STEP_MILLISECONDS);
-
-        self.total_sim_time_elapsed += context.delta();
+        self.core_hydraulic_updater.update(context);
+        self.physics_updater.update(context);
 
-        // Time to catch up in our simulation = new delta + time not updated last iteration
-        let time_to_catch = context.delta() + self.lag_time_accumulator;
+        // Here we update at the same rate as the sim or max time step if sim rate is too slow
+        self.update_max_fixed_step(&context);
```

Comment:
**thought**: The comment here and on the function itself are different. Consider removing this comment.